test/test_serving.py

```diff
@@ -57,6 +57,40 @@ def test_app_invoke(app):
     assert "application/json" == result.content_type
 
 
+def test_app_invoke_with_accept_any():
+    with patch.dict('os.environ', {'SAGEMAKER_DEFAULT_INVOCATIONS_ACCEPT': 'application/NPY'}):
```

Comment:
nit: should to use something that looks like valid mimetype even though test doesn't care (e.g. application/x-npy)